tests/Application/src/Entity/Channel.php

```diff
@@ -14,6 +14,6 @@
 #[ORM\Table(name: 'sylius_channel')]
 class Channel extends BaseChannel implements ChannelInterface
 {
-    use LowestPriceForDiscountedProductsVisibleAwareTrait;
     use LowestPriceForDiscountedProductsCheckingPeriodTrait;
+    use LowestPriceForDiscountedProductsVisibleAwareTrait;
```

Comment:
I would be for merging these two traits into one `LowestPriceForDiscountedProductsAwareTrait`